backend/daemon/storage/testdata/mintter-test-db-snapshot/VERSION

```diff
@@ -1 +1 @@
-2023-09-22.01
\ No newline at end of file
+2024-02-23.01
```

Comment:
No, don’t touch this file! @juligasa

This folder contains the “baseline” database which is the first ever schema we support migrations for. So on top of this database migrations are applied in a test to verify that the migrated schema matches the desired schema defined in schema.sql file. 

Basically, first you want to change schema.sql file to the new desired schema, and then create the necessary migration to move from the previous schema to the desired schema.

Reply:
See this for more info: https://github.com/MintterHypermedia/mintter/blob/b93b795e45b0fb0f180249d9558dd23462265250/backend/daemon/storage/migrations_test.go#L19

I now realize that this was not the most obvious place to find this information (should find a better home for it 😁).

Reply:
Oh yes I read through that but since I was getting an error in the tests saying 
```
--- Expected
                                +++ Actual
                                @@ -102,4 +102,5 @@
                                     author INTEGER REFERENCES public_keys (id),
                                -    resource INTEGER REFERENCES resources (id)
                                -, meta TEXT) WITHOUT ROWID
                                +    resource INTEGER REFERENCES resources (id),
                                +    meta TEXT
                                +) WITHOUT ROWID
                                 CREATE INDEX structural_blobs_by_author ON structural_blobs (author, resource) WHERE author IS NOT NULL
```
I though that I should update the Golden, But you are right is the baseline. The test should pass. However the reason why it does not pass seems like the generated schema after the ALTER table is not correct. I will drop tables and indexes and create them from scratch like in previous migrations.